zio-http/src/main/scala/zio/http/FormField.scala

```diff
@@ -35,11 +35,56 @@ sealed trait FormField {
   def contentType: MediaType
   def filename: Option[String]
 
-  final def valueAsString: Option[String] = this match {
+  /**
+   * Gets the value as a String, but only if it is a text or simple field. For
+   * binary fields it returns None.
+   */
+  final def stringValue: Option[String] = this match {
     case FormField.Text(_, value, _, _) => Some(value)
     case FormField.Simple(_, value)     => Some(value)
     case _                              => None
   }
+
+  /**
+   * Gets the value of this form field as a String. If it is a binary field, the
+   * value is interpreted as an UTF-8 byte stream.
+   */
+  final def asText: ZIO[Any, CharacterCodingException, String] = this match {
```

Comment:
I see now we have `stringValue`, which has some overlap with this one. Not sure if and how to unify the names.

Reply:
it was called `valueAsString` and I renamed it to `stringValue` :) Maybe we don't need it at all?